README.rst

```diff
@@ -12,6 +12,8 @@
 torchtext
 +++++++++
 
+CAUTION: This repository is no longer actively maintained by PyTorch. Use at your own risk. 
```

Comment:
Nit: maybe we want to communicate a slightly less scary message, like the one that was written on the [0.16 release notes](https://github.com/pytorch/text/releases/tag/v0.16.0) ?  But LGTM anyway, thanks @joecummings 

> As of September 2023 we have paused active development of TorchText because our focus has shifted away from building out this library offering. We will continue to release new versions but do not anticipate any new feature development as we figure out future investments in this space.